pkg/network/common/dns.go

```diff
@@ -400,3 +400,20 @@ func removeDuplicateIPs(ips []net.IP) []net.IP {
 
 	return uniqueIPs
 }
+
+func normalizeTTL(ttl time.Duration) time.Duration {
+
+	// The TTL is guaranteed to be valid for its duration, however, it's not
+	// guaranteed that the DNS server will cache it and always return the same
+	// list of IP addreses for the whole TTL.
+	// For records with large TTL (>30 min) we assume they will always return
+	// the same list of addresses and query them only once every 30 min. We'll
+	// assume rest of the records a TTL of at most 30 seconds.
+
+	if ttl < 30*time.Second {
+		return ttl
+	} else if ttl >= 1800*time.Second {
+		return 1800 * time.Second
```

Comment:
seriously, this is what gofmt expects? No wonder I can never figure out if it wants spaces or not...

anyway, `30 * time.Minute` would be clearer

Reply:
Yes, this is generated by gofmt :sweat_smile: